<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupplementalListWidget</class>
 <widget class="QWidget" name="SupplementalListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SupplementalListWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Supplemental List */


position: absolute;
left: 0%;
right: 0%;
top: 0%;
bottom: 0%;

background: #404040;</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>6</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 113px;
left: 28px;
top: 6px;
bottom: 6px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 16px;
display: flex;
align-items: center;

/* panel/section title */

color: rgba(255, 255, 255, 0.5);
background: rgba(255, 255, 255, 0.0001);</string>
   </property>
   <property name="text">
    <string>Supplemental List</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NoItem">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>36</y>
     <width>208</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
height: 15px;
left: 16px;
right: 16px;
top: 36px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 15px;
/* identical to box height */

display: flex;
align-items: center;

color: rgba(255, 255, 255, 0.45);
</string>
   </property>
   <property name="text">
    <string>There is no supplemental item.</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>36</y>
     <width>240</width>
     <height>98</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>9</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/* icon/resource-image */


position: absolute;
width: 24px;
height: 24px;
left: calc(50% - 24px/2);
top: calc(50% - 24px/2);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/SVG/svg/icon/toc-resource.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>6</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* icon/reading-order-add-to-list */


position: absolute;
width: 24px;
height: 24px;
left: calc(50% - 24px/2);
top: calc(50% - 24px/2);

border: 0px;
</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/common-add.svg</normaloff>:/SVG/svg/icon/common-add.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <zorder>listWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>label_NoItem</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_Add</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
